Source/WTF/wtf/text/ASCIILiteral.h

```diff
@@ -143,19 +140,20 @@ constexpr ASCIILiteral operator"" _s(const char* characters, size_t)
     return result;
 }
 
-constexpr std::span<const LChar> operator"" _span(const char* characters, size_t n)
+constexpr std::span<const LChar> operator"" _span(const char* characters, size_t)
 {
+WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
+    auto span = spanReinterpretCast<const LChar>(std::span { characters, std::char_traits<char>::length(characters) });
+WTF_ALLOW_UNSAFE_BUFFER_USAGE_END
```

Comment:
This warnings on macOS 15, but shouldn't because it's safe.

@geoffreygaren do we want to add `unsafeForgeSpan()` here, or make a different macro to denote safe patterns?

Reply:
unsafeForgeSpan() is the only option here because the pointer and length come to us as separate values, through an API that we can't change.